src/verilog.y

```diff
@@ -2912,23 +2912,17 @@ foperator_assignment<nodep>:	// IEEE: operator_assignment (for first part of exp
 	//UNSUP	BISONPRE_COPY(operator_assignment,{s/~f~/f/g})	// {copied}
 	;
 
-//UNSUPinc_or_dec_expression<nodep>:  // ==IEEE: inc_or_dec_expression
-//UNSUP	//			// Need fexprScope instead of variable_lvalue to prevent conflict
-//UNSUP		~l~exprScope yP_PLUSPLUS		{ $<fl>$=$<fl>1; $$ = $1+$2; }
-//UNSUP	|	~l~exprScope yP_MINUSMINUS		{ $<fl>$=$<fl>1; $$ = $1+$2; }
-//UNSUP	//			// Need expr instead of variable_lvalue to prevent conflict
-//UNSUP	|	yP_PLUSPLUS	expr			{ $<fl>$=$<fl>1; $$ = $1+$2; }
-//UNSUP	|	yP_MINUSMINUS	expr			{ $<fl>$=$<fl>1; $$ = $1+$2; }
-//UNSUP	;
+inc_or_dec_expression<nodep>:	// ==IEEE: inc_or_dec_expression
+	//			// Need fexprScope instead of variable_lvalue to prevent conflict
+		~l~exprScope yP_PLUSPLUS		{ $<fl>$=$<fl>1; $$ = new AstPostAdd($2, $1, $1->cloneTree(true), new AstConst($2, AstConst::StringToParse(), "'b1")); }
```

Comment:
My shuffle meant op1 is constant op2 is pre-increment ref, op3 is post-increment store-to value.

```suggestion
		~l~exprScope yP_PLUSPLUS		{ $<fl>$=$<fl>1; $$ = new AstPostAdd($2, new AstConst($2, AstConst::StringToParse(), "'b1"), $1, $1->cloneTree(true),); }
```